alcs-api/src/board/board.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Application } from '../application/application.entity';
+import { ApplicationService } from '../application/application.service';
+import { ServiceNotFoundException } from '../common/exceptions/base.exception';
+import { Board } from './board.entity';
+
+@Injectable()
+export class BoardService {
+  constructor(
+    @InjectRepository(Board)
+    private boardRepository: Repository<Board>,
+    private applicationService: ApplicationService,
+  ) {}
+
+  list() {
+    return this.boardRepository.find({
+      relations: ['statuses'],
+      order: {
+        statuses: {
+          order: 'ASC',
+        },
+      },
+    });
+  }
+
+  getApplicationsByCode(code: string) {
+    return this.applicationService.getAll({
+      board: {
+        code,
+      },
+    });
+  }
+
+  async changeBoard(fileNumber: string, code: string) {
+    const application = await this.applicationService.get(fileNumber);
+    if (!application) {
+      throw new ServiceNotFoundException('Failed to find application');
+    }
+
+    const board = await this.boardRepository.findOne({
+      where: {
+        code,
+      },
+      relations: ['statuses'],
```

Comment:
I think it is worth moving 'statuses' into defaultRelations as we did in other services

Reply:
Good call, done!